src/app/settings/SettingsPage.tsx

```diff
@@ -40,6 +40,8 @@ import SortOrderEditor, { SortProperty } from './SortOrderEditor';
 import Spreadsheets from './Spreadsheets';
 import { TroubleshootingSettings } from './Troubleshooting';
 
+const isMobile = 'ontouchstart' in document.documentElement;
```

Comment:
This isn't a super reliable wY to detect mobile. We usually use isPhonePortrait which detects the mobile layout

Reply:
I saw that but it works also on landscape orientation ?

Reply:
Yeah - if your intention is to detect places a streamdeck host couldn't run you'll have to do a lot more. 

Touchstart can be defined on touch-capable laptops/convertible tablets.

Reply:
I moved to `matchMedia` using the phone portrait media query